C/Ccluster/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Ccluster"
+version = v"1.1.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/rimbach/Ccluster/archive/refs/tags/v1.1.6.tar.gz", "85b866a5485403d8af944965c531645709c21ee0810adfe720258e5c509332e1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Ccluster*/
+if [[ ${target} == *musl* ]]; then      export CFLAGS="-D_GNU_SOURCE=1 -std=c99 -O2 -funroll-loops -g"; fi
+if [[ ${target} == *mingw* ]]; then sed -i -e "s#/lib\>#/$(basename ${libdir})#g" configure; extraflags=--build=MINGW${nbits}; fi
+./configure --prefix=$prefix --disable-pthread --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix --with-flint=$prefix --with-arb=$prefix ${extraflags}
```

Comment:
This is a bit easier to read:
```suggestion
cd $WORKSPACE/srcdir/Ccluster*/
if [[ ${target} == *musl* ]]; then
    export CFLAGS="-D_GNU_SOURCE=1 -std=c99 -O2 -funroll-loops -g"
elif [[ ${target} == *mingw* ]]; then
    sed -i -e "s#/lib\>#/$(basename ${libdir})#g" configure; extraflags=--build=MINGW${nbits}
fi
./configure --prefix=$prefix \
    --disable-pthread \
    --disable-static \
    --enable-shared \
    --with-gmp=$prefix \
    --with-mpfr=$prefix \
    --with-flint=$prefix \
    --with-arb=$prefix \
    ${extraflags}
```